test/test_core.py

```diff
@@ -5525,9 +5525,10 @@ def test_fstatat(self):
   @crossplatform
   @with_all_fs
   def test_stat_chmod(self):
-    if self.get_setting('NODERAWFS') and WINDOWS:
+    nodefs = '-DNODEFS' in self.emcc_args or '-DNODERAWFS' in self.emcc_args
+    if nodefs and WINDOWS:
```

Comment:
It looks like we use to run this test with `-DNODEFS` on windows, but not after this change?

Reply:
Ooops,  sorry I just saw that this change includes a change to the test (which presumably makes it no longer compatible with windows+nodefs).  lgtm.